lib/image.js

```diff
@@ -1,12 +1,84 @@
-"use strict";
-var Promise = require('bluebird');
-var fs = require('fs-extra');
+'use strict';
+var Cesium = require('cesium');
+var fsExtra = require('fs-extra');
 var path = require('path');
+var PNG = require('pngjs').PNG;
+var Promise = require('bluebird');
+
+var fsExtraReadFile = Promise.promisify(fsExtra.readFile);
 
-var fsReadFile = Promise.promisify(fs.readFile);
+var defaultValue = Cesium.defaultValue;
+var WebGLConstants = Cesium.WebGLConstants;
 
 module.exports = loadImage;
```

Comment:
Filename and function name don't match.  Shouldn't the file be called `loagImage`?